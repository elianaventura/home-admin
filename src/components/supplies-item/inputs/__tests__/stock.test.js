import { render } from "@testing-library/react";
import Stock from "@/components/supplies-item/inputs/stock";

describe('First', () => {
  it('renders passes test', () => {
    const { container } = render(<Stock title="Stock" />);
 
    expect(container).toMatchSnapshot();
  });
});
