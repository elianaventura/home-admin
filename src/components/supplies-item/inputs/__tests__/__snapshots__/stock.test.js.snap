// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stock renders passes test 1`] = `
<div>
  <div>
    <span
      class="stock-title"
    >
      Stock
    </span>
    <input
      class="stock-input"
      data-testid="stock-input"
      value="4"
    />
    <span>
      kg
    </span>
  </div>
</div>
`;
